model/src/main/kotlin/ScannerRun.kt

```diff
@@ -72,6 +73,12 @@ data class ScannerRun(
     @JsonSerialize(converter = ScanResultSortedSetConverter::class)
     val scanResults: Set<ScanResult>,
 
+    /**
+     * The scanners for each scanned package.
+     */
+    @JsonSerialize(converter = ScannersMapConverter::class)
```

Comment:
If the goal is to just sort the keys in the map by alphabetic string representation, using `@JsonPropertyOrder(alphabetic = true)` should be enough.

Reply:
Wouldn't ordering the values be needed as well to be deterministic, to allow for use with expected results?

Reply:
@fviernau 
@sschuberth 
Is there a special annotation to do that of should I again create a converter ?

Reply:
> Is there a special annotation to do that of should I again create a converter ?

If the values in a map should be sorted as well, I believe you need a converter again.